packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -302,16 +302,18 @@ const ruleset: any = {
     },
     // github issue https://github.com/Azure/azure-openapi-validator/issues/331
     // Get operation should return 200
-    // already have rule to check if operation returns non 2XX, it should mark it as 'x-ms-error-response' explicitly,
-    // so here on check if the 200 return '201','202','203'
+    // already have rule to check if operation returns non 2XX, it should mark it as 'x-ms-error-response' explicitly
+    // https://github.com/Azure/azure-openapi-validator/issues/549 
+    // GET can return 202 only if it is a polling action & has Location header defined. LroLocationHeader rule already checks if 202 response has Location header
+    // so here just check for non 200, 202 response codes i.e, '201','203','204'
     // RPC Code: RPC-Get-V1-01
```

Comment:
why not make it an inclusion list for 200, 202 instead of an exclusion list for 201, 203, 204?

Reply:
That way we get broader coverage

Reply:
thats a good point!
But we'd not be able to use the core function